consumer/types/host_types.go

```diff
@@ -1,8 +1,17 @@
 package types
 
+import (
+	"encoding/json"
+	"fmt"
+)
+
 const (
-	HostResourceType = "host"
-	HostReporterType = "hbi"
+	HostResourceType       = "host"
+	HostReporterType       = "hbi"
+	HostReporterInstanceID = "redhat.com"
```

Comment:
I feel like this came up in a PR with Arif lately (for HBI), i would expect this data to be unique for each host (specifically what system reported the host, the hosts API HREF and Console HREF) or do they not capture these either in actual data they have now

Reply:
Linked a thread in slack on this, but redhat.com is the preferred identifier here.